packages/gui/src/lib/stringify.ts

```diff
@@ -53,6 +55,16 @@ export function stringifyUnit(
     return value.value
   }
 
+  if (value.unit === 'theme') {
+    // resolve path from theme
+    const resolvedValue = theme && value.themePath && get(theme, value.themePath)
```

Comment:
Using lodash `get` here, but just seen theme-ui has it own get function. Think they mostly to the same thing except with maybe slightly different syntax. Will see if there's any advantage to either in clean up stage